classes/notification/NotificationAPIManager.php

```diff
@@ -333,7 +333,7 @@ public function getUserNotifications(): array
             }
         }
         //Event
-        if ($this->system->event != null) {
+        if (isset($this->system->event)) {
```

Comment:
Ah, we could alternatively fix this by defaulting `$event = null` in the System class. I suppose either is fine but IMO it's a little weird conceptually to ask if a class property is set (should always be set, just may or may not have a value)